run.py

```diff
@@ -32,15 +32,15 @@
 
         # Don't leak token
         if k == 'sm_cluster' and args_dict[k] is not None:
-            log.ODM_INFO('%s: True' % k)
+            log.ODM_INFO('%s: True' % k
```

Comment:
I think this parenthesis was removed by mistake?

Reply:
@pierotofy yes you are right. I'm sorry, this was my first open-source contribution. Do you want me to add the paranthesis?